Makefile

```diff
@@ -9,27 +9,21 @@ ifeq (, $(shell which richgo))
 $(warning "could not find richgo in $(PATH), run: go install github.com/kyoh86/richgo@latest")
 endif
 
-.PHONY: fmt lint test install_deps clean
+.PHONY: lint test clean
 
 default: all
 
-all: fmt test
```

Comment:
Removing `fmt` here means that when people run `make` locally, they will no longer get errors for formatting errors.  Considering the linter takes less tan 2 seconds to run, I suggest running it for `make all`.
If someone does not want to run it, they can do `make test` directly.

What do you think?

Reply:
`fmt` is removed here because the Makefile target `fmt` is removed in this PR. We can add `lint` in this target, but that will require contributors to have golangci-lint installed before running `make`. I think it is better not to require having golangci-lint in order to `make` cobra. At the same time, it does not make sense to have an specific make target for one single linter but not for others. Overall, the users/developers interested in contributing code will always want to `make lint test clean`.

Reply:
When I started contributing I would do `make` and think everything was ok.  It was only later that I realized this did not include the linters, so now I run `make lint` manually.

But I agree that making `golangc-lint` a requirement may be annoying.  But it will also be annoying to only notice the bad formatting once CI is run...

I'm not sure what is best...

Reply:
IMHO modifying the sources and building cobra are not directly related to contributing code back to the upstream. That is, users might want to use and modify this open source tool regardless of the linting rules enforced in this repository. From this point of view, the requirement/need to use golangci-lint belongs to contributing guidelines: https://github.com/spf13/cobra/blob/main/CONTRIBUTING.md. Everyone should read the contributing guidelines before submitting a PR to a project.